src/components/download/__tests__/DownloadButton.spec.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { DownloadButton } from '../DownloadButton';
+
+describe('DownloadButton', () => {
+    const renderComponent = props =>
+        shallow(
+            <DownloadButton
+                classes={{}}
+                toggleDownloadDialog={() => null}
+                {...props}
+            />
+        );
+
+    it('Should render a download button', () => {
+        const wrapper = renderComponent();
+        expect(wrapper.find('WithStyles(Button)').exists()).toBe(true);
+        expect(
+            wrapper
+                .find('WithStyles(Button)')
+                .render()
+                .text()
+        ).toEqual('Download');
+    });
+
+    it('Should render a download dialog', () => {
+        const wrapper = renderComponent();
+        expect(
+            wrapper.find('Connect(WithStyles(DownloadDialog))').exists()
```

Comment:
This could be extended to check if passing different props shows and hides the DIalog.  With Enzyme you can inspect props with `wrapper.find('something').prop('open')` and test that it is `false` by default and `open` when a different prop is passed or the button is clicked (depending on whether or not Redux is involved)

Reply:
This is checked in DownloadDialog.spec.js